webapp/src/features/intersections/notifications/notifications-table.tsx

```diff
@@ -267,15 +255,18 @@ export const NotificationsTable = (props: { simple: Boolean }) => {
             sx={{
               m: -1,
               mt: 3,
+              pb: 1,
             }}
           >
-            <Grid2 container justifyContent="left" spacing={3}>
+            <Grid2 container justifyContent="right" spacing={3}>
               <Grid2>
                 <Button
                   sx={{
                     m: 1,
+                    mr: 3,
                   }}
-                  variant="contained"
+                  variant="outlined"
```

Comment:
Check the contrast on this one - the outlined variant is a little hard to read against the dark background

Reply:
![image](https://github.com/user-attachments/assets/72e53c49-5335-45c1-9c42-4518df74603f)

Reply:
Updated to use a lighter color for the outlined button.